src/Agent/NewRelic/Agent/Core/Api/AgentApiImplementation.cs

```diff
@@ -502,7 +502,7 @@ public void IgnoreApdex()
         ///   &lt;body&gt;
         ///   ...
         /// </code></example>
-        public string GetBrowserTimingHeader()
+        public string GetBrowserTimingHeader(string? nonce)
```

Comment:
Instead of using optional parameter, we recommend introducing the `GetBrowserTimingHeader(string nonce)` overloading method. Inside the old `GetBrowserTimingHeader()` method, we just have to call `GetBrowserTimingHeader(null)`. We try to avoid using optional parameter as it would cause future test complication.

Reply:
Makes sense; it also prevents a binary-breaking change.

Reply:
I added an overload for the public API; I can do the same here.

Reply:
that would be awesome! 👍

Reply:
did u already commit the change ? I have not since any changes.

Reply:
Oh, sorry, that was confusing.

I meant that in the initial PR, there was an overload in the public API, just not in the implementation. I can make them both the same.

Reply:
Pushed now.